duckduckgo/iOS
# Update Duck Address copy

Task/Issue URL: https://app.asana.com/0/414235014887631/1203067868196911/f
Tech Design URL:
CC:

**Description**:

This PR updates the Duck Address generation copy to "Generate Private Duck Address".

<details>
  <summary>👉 Click here for screenshots! 👈</summary>
  
  ![Action Sheet](https://user-images.githubusercontent.com/183774/194724975-0b508053-2ca5-493d-bea1-e3fd22127f47.png)
  ![Browsing Menu](https://user-images.githubusercontent.com/183774/194724982-4156b398-47ca-4e07-a9f7-0adbd3e502da.png)
</details>



**Steps to test this PR**:
1. Check that the browsing menu says "Generate Private Duck Address"
2. Check that the email action sheet says "Generate Private Duck Address"
3. Check that the translated copy changes look okay - this is my first time making a localization change



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
